doc/installation/ubuntu.rst

```diff
@@ -8,7 +8,7 @@ Ubuntu Packages
 
 There are ready made packages for Ubuntu.
 Up to privacyIDEA 2.23 packages are available for 
-Ubuntu 14.04 LTS and 16.04 LTS [#ubuntu1604]_. 
```

Comment:
14.04 packages are still available at launchpad.
So in fact they could be installed there. I think we can leave this in here.

Reply:
will do

Reply:
Done in e4a50d61